src/gleam/erlang/process.gleam

```diff
@@ -107,10 +102,8 @@ fn raw_send(a: Pid, b: message) -> DoNotLeak
 /// send(subject, "Hello, Joe!")
 /// ```
 ///
-pub fn send(subject: Subject(message), message: message) -> Nil {
-  raw_send(subject.owner, #(subject.tag, message))
-  Nil
-}
+@external(erlang, "gleam_erlang_ffi", "send")
+pub fn send(subject: Subject(message), message: message) -> Nil
```

Comment:
How come this has changed?

Reply:
I just thought I would make it more simple. Instead of using raw_send which returns a `DoNotLeak` I call own `gleam_erlang_ffi:send` which pattern match the subject and returns `nil` directly.
But maybe I missed something? Do you need `DoNotLeak` if the erlang function returns `nil`

Reply:
Having more code in Gleam and verified by Gleam is better!

Reply:
OK, I will revert this back to original.